### PR TITLE
Append commit sha1 to version name

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
It is difficult to find the commit using the current version name.
So, append the current commit's SHA1 for better debugging.

e.g. `7539-SNAPSHOT(2 +5 -1) a345bc3f96dc369c32eb1b540890ac3fb4eba7b2`

@gino-m PTAL?

### CODE REVIEW

File: gnd/build.gradle [hunk 1]
```diff
@@ -56,7 +56,7 @@ android {
         // fix before increasing this value.
         targetSdkVersion 26
         versionCode gitVersioner.versionCode
-        versionName gitVersioner.versionName
+        versionName gitVersioner.versionName + " " + gitVersioner.currentSha1
```
Comment (gino-m) [inline gnd/build.gradle:59]: Is there a way to get the shortened commit ID instead? From the command line the latest commit can be shown with `git show -s --format=%h`.

**Review by gino-m — CHANGES_REQUESTED** [review]

File: gnd/build.gradle (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/build.gradle:59]: The git versioner plugin doesn't provide that out of the box. I'll try doing it directly like you suggested. Maybe we can remove this dependency too.

Comment (gino-m) [reply gnd/build.gradle:59]: The short version is just the first 7 characters of the commit hash. Is it possible to truncate to only include those?

**Review by gino-m — DISMISSED** [review]

File: gnd/build.gradle (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/build.gradle]: Done

Comment (shobhitagarwal1612) [reply gnd/build.gradle]: sample output : `7571-version+3 6714acd75`

**Review by gino-m — APPROVED** [review]
Cool trick, learning a lot from these PRs, thanks again!

**Pull request merged**